tests/client_test.py

```diff
@@ -343,7 +347,6 @@ def test_get_type_invalid_version(self):
 
 
 class MetadataInterceptorTest(TestCase):
-
```

Comment:
It's actually recommended to have a blank line here:
https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Oh great, I added this back in and also in the other classes where it didn't exist.